addons/website_sale/static/src/snippets/s_dynamic_snippet_products/000.js

```diff
@@ -145,7 +145,7 @@ const DynamicSnippetProductsCard = WebsiteSale.extend({
 
             const inputPT = document.createElement('input');
             inputPT.setAttribute('name', 'product_template_id');
-            inputPT.classList.add('product_template_id');  // how can this logic be so ugly??? T_T
```

Comment:
@livaios this was only used to retrieve the product template directly inside `_handleAdd` right? (I updated _handleAdd to no longer rely on the `class` in this commit)

Reply:
You can say I'm the culprit behind this part of your commit, I assume my 'how can this logic be so ugly??' :D